core/pkg/policyhandler/handlenotification.go

```diff
@@ -80,38 +82,68 @@ func getDesignator(policyIdentifier []cautils.PolicyIdentifier) *armotypes.Porta
 }
 
 func setCloudMetadata(opaSessionObj *cautils.OPASessionObj) {
-	cloudMetadata := getCloudMetadata(opaSessionObj)
-	if cloudMetadata == nil {
+	iCloudMetadata := getCloudMetadata(opaSessionObj, k8sinterface.GetConfig())
```

Comment:
`iCloudMetadata`?

Reply:
Yes, the `getCloudMetadata` function returns an interface